models/classes/TaoLtiSession.php

```diff
@@ -28,27 +26,33 @@
 use oat\taoLti\models\classes\ResourceLink\LinkService;
 use oat\taoLti\models\classes\user\LtiUserInterface;
 
-/**
- * The TAO layer ontop of the LtiSession
- *
- * @access public
- * @author Joel Bout, <joel@taotesting.com>
- * @package taoLti
- */
 class TaoLtiSession extends common_session_DefaultSession
 {
-    /**
-     * @var core_kernel_classes_Resource
-     */
-    private $ltiLink = null;
+    private const VERSION_LTI_1P1 = '1.1';
+    private const VERSION_LTI_1P3 = '1.3';
+
+    /** @var string */
+    protected $linkId = null;
+
+    /** @var string */
+    protected $version = self::VERSION_LTI_1P1;
 
     public function __construct(LtiUserInterface $user)
     {
         parent::__construct($user);
     }
 
+    public static function version1p3(LtiUserInterface $user): self
```

Comment:
Can we consider here a different name that would be a bit more self-explanatory, might be `createWithVerstion1p3`  or so on